staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation_test.go

```diff
@@ -1688,12 +1688,12 @@ func TestCostEstimation(t *testing.T) {
 					"before": strType,
 					"after":  strType,
 				})
-				objType = withRule(objType, "self.str.replace(self.before, self.after) == 'does not matter'")
+				objType = withRule(objType, "self.str.replace(self.before, self.after) == '0123456789'")
```

Comment:
any explanation for why this test changed or why the new value is better / more correct?

Reply:
Good question. This is the only test that exercised the behavior being fixed.  But it failed to catch it.

The estimator calculates the cost of the equals check based on the `min(length(LHS), length(RHS))` of the equal operation.

Before fixing the bug it was doing `min(length(MAX_INT), length("does not matter"))` and so the expected cost values were based on the length of "does not matter".  When I fixed the bug it changed the expected values because it became `length(<expected replace result size>)` became less than "does not matter".

I actually don't need to keep this string change, I modified it when testing my assumptions locally, I'll revert it.  The expected values will still change.